locations/spiders/Amazons.py

```diff
@@ -0,0 +1,130 @@
+import scrapy
+from scrapy.linkextractors import LinkExtractor
+from scrapy.spiders import CrawlSpider, Rule
+import  requests
+from locations.items import  GeojsonPointItem
+
+User_Agent = "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2403.157 Safari/537.36"
+
+
+days = ["Monday","Tuesday","Wednesday","Thursday","Friday","Saturday","Sunday"]
+days_short =["Mon","Tue","Wed","Thur","Fri","Sat","Sun"]
+days_want = ["Mo","Tu","We","Th","Fr","Sa","Su"]
+days_dict = {i:n for n,i in enumerate(days_want)}
+days_rev_dict = {n:i for n,i in enumerate(days_want)}
+class AmazonsSpider(CrawlSpider):
+    name = 'Amazons'
+    allowed_domains = ['amazon.com']
+    # start_urls = ['']
+    user_agent = (
+        User_Agent
+    )
+
+    def start_requests(self):
+        yield scrapy.Request(
+            url="https://www.amazon.com/find-your-store/b/?node=17608448011",
+            headers={"User-Agent":self.user_agent}
+        )
+
+    rules = (
+        Rule(LinkExtractor(restrict_xpaths="//p/a"),
+             callback='parse_item',
+             follow=True,
+             process_request="set_user_agent"),
+    )
+    def set_user_agent(self, request):
+        request.headers["User-Agent"] = self.user_agent
+        return request
+
+    def parse_item(self, response):
+        item = {}
+        openhours = []
+        linker = []
+        locations = None
+        phela = None
+        naam = response.xpath("//h2/text()").get()
```

Comment:
Please use English variable names.
```suggestion
        name = response.xpath("//h2/text()").get()
```